managed/utils/collectors/collectors.go

```diff
@@ -42,3 +42,20 @@ func FilterOutCollectors(prefix string, args, disabledCollectors []string) []str
 	}
 	return enabledArgs
 }
+
+// DisableDefaultEnabledCollectors returns CLI arguments to disable default enabled collectors based on input.
+// DefaultCollectors and disabledCollectors should be collector names without prefix.
+// Result will be returned with prefix.
+func DisableDefaultEnabledCollectors(prefix string, defaultCollectors []string, disabledCollectors []string) []string {
```

Comment:
Considering we always use `--no-collector` as prefix for disabling collectors, should we just make it a constant and drop the prefix parameter?
```suggestion
func DisableDefaultEnabledCollectors(defaultCollectors []string, disabledCollectors []string) []string {
```